packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -41,7 +41,9 @@ const POPOVER_PROPS = {
 
 function CopyMenuItem( { blocks, onCopy } ) {
 	const ref = useCopyToClipboard( () => serialize( blocks ), onCopy );
-	return <MenuItem ref={ ref }>{ __( 'Copy' ) }</MenuItem>;
+	const copyMenuItemLabel =
+		blocks.length > 1 ? __( 'Copy blocks' ) : __( 'Copy block' );
```

Comment:
Should never attempt to manually wrangle plurals in i18n. Instead, use `_n`:

```suggestion
	const copyMenuItemLabel = _n( 'Copy block', 'Copy blocks', blocks.length );
```

Different languages handle plurals and counting in ways that exceed our imagination, like:
- singular (1), dual (2), plural (3+)
- no inflection at all
- special zero case
- counting particles
- etc.

Reply:
I had no idea abou `_n`, now I do!